src/main/java/frc/robot/RobotContainer.java

```diff
@@ -117,19 +117,19 @@ public RobotContainer() {
         if (driveSubsystem instanceof BaseSwerveSubsystem) {
             final BaseSwerveSubsystem swerveSubsystem = (BaseSwerveSubsystem) driveSubsystem;
 
-            autonChooser.addOption("Straight-line path", new StraightLinePath(swerveSubsystem));
-            autonChooser.addOption("High rotation straight-line path", new HighRotationLinePath(swerveSubsystem));
-            autonChooser.addOption("Rotating S-curve", new RotatingSCurveAutonSequence(swerveSubsystem));
-            autonChooser.addOption("Box auton", new BoxAutonSequence(swerveSubsystem));
-            autonChooser.addOption("GRT path", new GRTAutonSequence(swerveSubsystem));
-
             autonChooser.addOption("Red top auton", new RedTopAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
             autonChooser.addOption("Red balance auton", new RedBalanceAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
             autonChooser.addOption("Red bottom auton", new RedBottomAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
 
             autonChooser.addOption("Blue top auton", new BlueTopAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
-            // autonChooser.addOption("Blue balance auton", new BlueBalanceAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
+            autonChooser.addOption("Blue balance auton", new BlueBalanceAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
             autonChooser.addOption("Blue bottom auton", new BlueBottomAuton(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
```

Comment:
perhaps we can just add two Base(Top/Balance/Bottom)Auton classes, Red and Blue, for each path, instead of having explicit classes for each one? if the only difference is in passing a true/false we can probably do it here

Reply:
or perhaps we can have two different choosers: auton path & red/blue

Reply:
Two choosers won't work since the classes need the boolean when they're constructed, but I can def see the base autons no longer being abstract.